.rhcicd/clowdapp.yaml

```diff
@@ -140,6 +140,40 @@ objects:
                         secretKeyRef:
                           name: policies-ui-backend-db
                           key: ${DB_SECRET_PASSWORD_KEY}
+  - apiVersion: metrics.console.redhat.com/v1alpha1
+    kind: FloorPlan
+    metadata:
+      name: policies-backend-hms
+    spec:
+      database:
+        secretName: ${FLOORIST_DB_SECRET_NAME}
+      objectStore:
+        secretName: ${FLOORIST_HMS_BUCKET_SECRET_NAME}
+      logLevel: ${FLOORIST_LOGLEVEL}
+      suspend: ${{FLOORIST_SUSPEND}}
+      disabled: ${{FLOORIST_DISABLED}}
+      queries:
+        - prefix: hms_analytics/policies/status
+          query: >-
+            SELECT
+              P.id,
+              P.name,
```

Comment:
Name isn't necessary to be collected, or is it?

Reply:
yeah I don't think so